src/main/java/net/pms/dlna/DLNAMediaInfo.java

```diff
@@ -958,22 +1092,60 @@ public void parse(InputFile inputFile, Format ext, int type, boolean thumbOnly,
 
 						if (sz > 0 && !net.pms.PMS.isHeadless()) {
 							BufferedImage image = ImageIO.read(new ByteArrayInputStream(thumb));
```

Comment:
Cool, that's good news. Maybe it was added in error. The final months of PMS were littered with massive commits with little explanation of the changes contained, and we ended up reverting a lot of them, so this may be an example of one that slipped through the cracks.